daemon/cmd/daemon.go

```diff
@@ -611,6 +611,9 @@ func NewDaemon(ctx context.Context, epMgr *endpointmanager.EndpointManager, dp d
 		}
 	}
 
+	// Given BPF datapath is up, we can now remove old maps from version upgrade.
```

Comment:
This is clearly after `d.init()` above for the base prog initialization, but I suspect we don't yet know whether the endpoints have been regenerated? Is this meaningful?

Reply:
@joestringer I don't think we need this part anymore. We only need the second commit that changes `cilium-map-migrate`. Daniel mentioned he needed to investigate a failure in the CI, but doesn't have the cycles right now. I'll mark as draft in the meantime.